src/components/app-configure/auth/WebWallet.vue

```diff
@@ -95,6 +95,7 @@ function handleUIModeUpdate(value: boolean) {
               <VSwitch
                 :model-value="walletUIMode"
                 class="ui-mode-switch"
+                :disabled="walletUIMode"
```

Comment:
can we change the name of the `walletUIMode` variable, it doesn't sound like a boolean type variable. example: `isNoUIMode` or `isUIMode`

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/91/commits/04e2453eaec58f5030749229f8c43b4530eba127